kong/clustering/init.lua

```diff
@@ -121,31 +110,8 @@ function _M:init_dp_worker(plugins_list)
       return
     end
 
-    local config_proto, msg
-    if not kong.configuration.legacy_hybrid_protocol then
-      config_proto, msg = check_protocol_support(self.conf, self.cert, self.cert_key)
-      -- otherwise config_proto = nil
-    end
-
-    if not config_proto and msg then
-      ngx_log(ngx_ERR, _log_prefix, "error check protocol support: ", msg)
-    end
-
-    ngx_log(ngx_DEBUG, _log_prefix, "config_proto: ", config_proto, " / ", msg)
-
-    local data_plane
-    if config_proto == "v0" or config_proto == nil then
-      data_plane = "kong.clustering.data_plane"
-
-    else -- config_proto == "v1" or higher
-      data_plane = "kong.clustering.wrpc_data_plane"
-    end
-
-    self.child = require(data_plane).new(self.conf, self.cert, self.cert_key)
-
-    if self.child then
-      self.child:init_worker(plugins_list)
-    end
+    self.child = require("kong.clustering.wrpc_data_plane").new(self.conf, self.cert, self.cert_key)
```

Comment:
`require`s at the top of the file, please (unless there's a really strong reason to do this here)

Reply:
Agreed, though I think there's an argument to be made about lazy instantiation since the DP module need not be loaded in all contexts (that's a discussion for another day) :P

The real answer is that this was just done to keep the diff small. I will leave a note to follow up with a code-tidying PR shortly :+1: